rust_src/src/image.rs

```diff
@@ -13,4 +14,14 @@ pub fn imagep(spec: LispObject) -> bool {
     unsafe { valid_image_p(spec) }
 }
 
+/// Initialize image library implementing image type TYPE.
+/// Return non-nil if TYPE is a supported image type.
+/// If image libraries are loaded dynamically (currently only the case on
+/// MS-Windows), load the library for TYPE if it is not yet loaded, using
+/// the library file(s) specified by `dynamic-library-alist'.
+#[lisp_fn]
+pub fn init_image_library(r#type: LispObject) -> bool {
+    unsafe { lookup_image_type(r#type) }
```

Comment:
`lookup_image_type` is declared `static` in the C code. `static` marks the function as only accessible to that C file. The name is not exposed. This is why Rust cannot see it. We often need to remove the `static` modifier and then copy the function definition into a header so it will be visible to Rust.